kouncil-backend/src/main/java/com/consdata/kouncil/topic/TopicService.java

```diff
@@ -48,6 +52,9 @@ public class TopicService {
     private final MessagesHelper messagesHelper;
     private static final int RESEND_MAX_POLL_RECORDS = 100;
 
+    @Value("${kouncil.topics.resend-headers-to-keep}")
+    private String[] resendHeadersToKeep;
```

Comment:
Maybe one service with all custom configuration? + validation @Validated from spring

Reply:
Imo that's too much for now, especially if this is temporary solution and passing empty list of headers to keep makes sense (validation is not useful)